Lib/test/test_xml_etree_c.py

```diff
@@ -234,6 +234,27 @@ def test_element_with_children(self):
         self.check_sizeof(e, self.elementsize + self.extra +
                              struct.calcsize('8P'))
 
+
+@unittest.skipUnless(cET, 'requires _elementtree')
```

Comment:
Why is this needed? From the bug, it sounds like the warning should be the same whether or not the C implementation is available.

Reply:
Thanks, I now realize I can move this test to the suite for the Python implementation, since the C version runs those tests also.